packages/site/src/layout/BaseView.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box } from "@jobber/components";
+import React from "react";
+
+export function BaseView({ children }: { readonly children: React.ReactNode }) {
```

Comment:
Love that you created this component!

This is a nit, not blocking, but we can make use of `PropsWithChildren` to simplify the type definitions. So we can have this instead:

```
export function BaseView({ children }: PropsWithChildren<{}>) {
  return <div style={{ display: "flex", height: "100dvh" }}>{children}</div>;
}
```

It's a utility type that automatically includes `children` as type React.ReactNode. Again, not a blocker, just something I learned a while ago.

Reply:
Ah good call, I have seen that in the code but forgot to reach for it when building this. Definitely an improvement and now added